build

```diff
@@ -125,10 +125,7 @@ make_docs() {
 }
 
 assert_no_compile_time_only_deps() {
-    if [ "$("$FIND" "_build/$PROFILE/rel/emqx/lib/" -maxdepth 1 -name 'gpb-*' -type d)" != "" ]; then
-        echo "gpb should not be included in the release"
-        exit 1
-    fi
+    :
```

Comment:
Talking to @zmstone , we decided to lift this restriction on `gpb` because it is actually [LGPL](https://github.com/tomas-abrahamsson/gpb/blob/5557e3117235b7df70a302c4f18267ec95777803/COPYING.LIB#L1-L17) and not GPLv3.